docs/maui/views/popup-service.md

```diff
@@ -147,9 +170,17 @@ public class MyViewModel : INotifyPropertyChanged
         this.popupService = popupService;
     }
 
-    public void DisplayPopup()
+    public async Task DisplayPopup()
     {
-        this.popupService.ShowPopup<UpdatingPopupViewModel>(onPresenting: viewModel => viewModel.Name = "Shaun");
+        var queryAttributes = new Dictionary<string, object>
+        {
+            ["Name"] = "Shaun"
+        };
+
+        await this.popupService.ShowPopupAsync<NamePopupViewModel>(
+            Shell.Current,
```

Comment:
I'm totally fine with it, but I believe that makes sense to point out that some people would argue that the VM shouldn't know nothing about page/UI, so `Shell.Current` here would break the MVVM

Reply:
How would you avoid referring to `Shell.Current` in a view model? Use some form of navigation service implementation? If so then I am OK with just referring to `Shell.Current` here as the MVVM purists will likely have a solution to this already and I fear abstracting it away will only complicate the documentation